services/director-v2/requirements/_base.txt

```diff
@@ -1,41 +1,55 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.9
+# by the following command:
 #
 #    pip-compile --output-file=requirements/_base.txt --resolver=backtracking --strip-extras requirements/_base.in
 #
 aio-pika==8.2.4
-    # via -r requirements/../../../packages/service-library/requirements/_base.in
+    # via
+    #   -r requirements/../../../packages/service-library/requirements/_base.in
+    #   -r requirements/_base.in
 aiocache==0.11.1
-    # via -r requirements/../../../packages/simcore-sdk/requirements/_base.in
+    # via
+    #   -r requirements/../../../packages/simcore-sdk/requirements/_base.in
+    #   -r requirements/_base.in
 aiodebug==2.3.0
-    # via -r requirements/../../../packages/service-library/requirements/_base.in
+    # via
+    #   -r requirements/../../../packages/service-library/requirements/_base.in
+    #   -r requirements/../../../packages/simcore-sdk/requirements/../../../packages/service-library/requirements/_base.in
 aiodocker==0.19.1
     # via -r requirements/_base.in
 aiofiles==0.8.0
-    # via -r requirements/../../../packages/service-library/requirements/_base.in
+    # via
+    #   -r requirements/../../../packages/service-library/requirements/_base.in
+    #   -r requirements/../../../packages/simcore-sdk/requirements/_base.in
 aiohttp==3.8.3
     # via
     #   -r requirements/../../../packages/simcore-sdk/requirements/_base.in
     #   aiodocker
     #   dask-gateway
 aiopg==1.3.3
-    # via -r requirements/../../../packages/simcore-sdk/requirements/_base.in
+    # via
+    #   -r requirements/../../../packages/simcore-sdk/requirements/_base.in
+    #   -r requirements/_base.in
 aioredis==2.0.1
     # via aiocache
 aiormq==6.4.2
     # via aio-pika
 aiosignal==1.2.0
     # via aiohttp
 alembic==1.8.1
-    # via -r requirements/../../../packages/postgres-database/requirements/_base.in
+    # via
+    #   -r requirements/../../../packages/postgres-database/requirements/_base.in
```

Comment:
Please check that you are using the write version of pip-tools.
To be sure, I recommend you to use the one prepared in the requirements image:
```
cd requirements/tools
make build-nc
make shell
cd ...
cd services/director-v2/requirements
make touch
make reqs target=async-dns
```

Reply:
Those are the exact commands that I am using. I don't know why so much stuff changes.